app/views/admin/statistics_announcements/show/_main.html.erb

```diff
@@ -0,0 +1,137 @@
+        text: "Release date",
+        heading_level: 3,
+        font_size: "m",
+        margin_bottom: 2,
+      } %>
+      <% if @statistics_announcement.cancelled? %>
+       <p>
+         <del class="govuk-body"><%= @statistics_announcement.release_date.to_fs(:date_with_time) %></del>
+         <span class="govuk-body"> (cancelled)</span>
+       </p>
+      <% else %>
+        <p class="govuk-body"> <%= @statistics_announcement.release_date.to_fs(:date_with_time) %> </p>
+      <% end %>
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Delivery date",
+        heading_level: 3,
+        font_size: "m",
+        margin_bottom: 2,
+      } %>
+      <p class="govuk-body">
+        <span><%= @statistics_announcement.display_date %></span>
+        <span>(<%= @statistics_announcement.confirmed? ? 'confirmed' : 'provisional' %>)</span>
+      </p>
+    </div>
+  </section>
+
+  <section class="app-view-statistics-announcements-summary__section">
+    <div class="govuk-body">
+      <%= render "govuk_publishing_components/components/heading", {
+        text: "Connected document",
+        heading_level: 2,
+        font_size: "l",
+        margin_bottom: 3,
+      } %>
+      <p>
+        <% if @statistics_announcement.publication %>
+          <%= link_to("Change connected document",
+                      admin_statistics_announcement_publication_index_path(@statistics_announcement),
+                      class: "govuk-link ,govuk-!-margin-bottom-8") %>
+        <table class="govuk-table">
+          <tr>
+            <td><%= @statistics_announcement.publication.title %></td>
+            <td><%= link_to("View",
+                            admin_statistics_announcement_publication_index_path(@statistics_announcement),
+                            class: "govuk-link") %></td>
+          </tr>
+        </table>
+      <% else %>
+      <span><%= link_to("Add existing document",
+                        admin_statistics_announcement_publication_index_path(@statistics_announcement),
+                        class: "govuk-link") %></span>
+        <span> or </span>
+        <span><%= link_to("create new document",
+                          new_admin_publication_path(statistics_announcement_id: @statistics_announcement),
+                          class: "govuk-link") %></span>
+      <% end %>
+      </p>
+    </div>
+  </section>
+
+  <section class="app-view-statistics-announcements-summary__section">
+    <%= render "govuk_publishing_components/components/heading", {
+      text: "Topic taxonomy tags",
+      heading_level: 2,
+      font_size: "l",
+      margin_bottom: 3,
+    } %>
+
+    <p class="govuk-body">
+      <%= link_to(@edition_taxons.any? ? "Change tags" : "Add tags",
+                  edit_admin_statistics_announcement_tags_path(@statistics_announcement.id),
+                  class: 'govuk-link') %>
+    </p>
+
+    <% @edition_taxons.map(&:full_path).each do |tag_path| %>
```

Comment:
i suspect this is pretty close to the implementation in the other summary page. I wonder if we can dry this out a bit. It feels like there's an opportunity to reuse the same css across both pages and create a shard partial for som of this stuff